tesseract/tesseract_scene_graph/src/srdf_model.cpp

```diff
@@ -0,0 +1,760 @@
+/**
+ * @file srdf_model.cpp
+ * @brief Parse srdf xml
+ *
```

Comment:
Would it be a bad idea to split this into 2 cpp files? We could have one for private methods and one for public. You could even have one for getters/setters if you wanted to get crazy.

Reply:
Broke up into separate files similar to the urdf parser.